### DISCUSSION TITLE
Quarkus 3.9.3 released - Maintenance release

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#40004] [Quarkus Blog/Website]
# Quarkus 3.9.3 released - Maintenance release

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-9-3-released/

**Comment by gediminasre** [comment]
Hello, how long should `quarkus update` usually take? I have updated manually from v3.5.0 to v3.8.3 and now I am trying to run `quarkus update`, but it never ends. The longest I waited was maybe ~1hour.

Here are the logs I see and it does not change:

$ quarkus update
To honour the JVM settings for this build a single-use Daemon process will be forked. For more on this, please refer to https://docs.gradle.org/8.6/userguide/gradle_daemon.html#sec:disabling_the_daemon in the Gradle documentation.
Daemon will be stopped at the end of the build

> Configure project :app
The 'sonarqube' task depends on compile tasks. This behavior is now deprecated and will be removed in version 5.x. To avoid implicit compilation, set property 'sonar.gradle.skipCompile' to 'true' and make sure your project is compiled, before analysis has started.
The 'sonar' task depends on compile tasks. This behavior is now deprecated and will be removed in version 5.x. To avoid implicit compilation, set property 'sonar.gradle.skipCompile' to 'true' and make sure your project is compiled, before analysis has started.  

> Task :quarkusUpdate
quarkusUpdate is experimental, its options and output might change in future versions
Detected project Java version: 17
Detected project Java version: 17
Instructions to update this project from '3.8.3' to '3.9.3':
No Quarkus extensions were found among the project dependencies
Resolved io.quarkus:quarkus-updates-recipes:1.0.15 with 1 recipe(s) to update from 3.8.3 to 3.9.3 (initially made for OpenRewrite GRADLE plugin version: 6.10.0) 
OpenRewrite recipe generated: C:\Users\GEDIMI~1.RED\AppData\Local\Temp\quarkus-project-recipe-11318085857365946396.yaml



 ------------------------------------------------------------------------
Executing:
C:\Users\user\IdeaProjects\ssp-service\gradlew.bat --console plain --stacktrace --init-script C:\Users\GEDIMI~1.RED\AppData\Local\Temp\openrewrite-init278914918606348932gradle rewriteRun

Starting a Gradle Daemon, 1 busy and 2 incompatible and 7 stopped Daemons could not be reused, use --status for details

**Reply by ewsachse** [reply]
I am seeing the same issue.   It starts up gradle and then just sits there.
